package.json

```
@@ -1,12 +1,12 @@
 {
   "name": "renderkid",
-  "version": "2.0.5",
+  "version": "2.0.6",
   "description": "Stylish console.log for node",
   "main": "lib/RenderKid.js",
   "dependencies": {
-    "css-select": "^2.0.2",
+    "css-select": "^4.1.2",
```

To mitigate CVE-2021-33587 I suggest to use:
```suggestion
    "css-select": "^4.1.3",
```